platform/marvell-arm64/linux/Makefile

```diff
@@ -0,0 +1,10 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+LINUX_KERNEL_MRVL_URL = https://github.com/Marvell-switching/sonic-marvell-binaries/raw/master/arm64/kernel/$(LINUX_KERNEL)
```

Comment:
we'd like the kernel to have source code and incorporated into sonic-linux-kernel repo.

Reply:
Ok will work on doing changes sonic-linux-kernel